pkgs/development/libraries/mesa/default.nix

```diff
@@ -116,6 +117,10 @@ self = stdenv.mkDerivation {
     "-Dgallium-nine=${boolToString enableGalliumNine}" # Direct3D in Wine
     "-Dosmesa=${boolToString enableOSMesa}" # used by wine
     "-Dmicrosoft-clc=disabled" # Only relevant on Windows (OpenCL 1.2 API on top of D3D12)
+
+    # OpenCL
+    "-Dgallium-opencl=icd" # Enable the gallium OpenCL frontend
+    "-Dclang-libdir=${llvmPackages.clang-unwrapped.lib}/lib"
   ] ++ optionals stdenv.isLinux [
```

Comment:
maybe opencl stuff a bit below, under `isLinux`?

Reply:
Good point. I moved the options to the Linux-specific ones.